src/main/java/com/google/sps/model/GeocodingClientImpl.java

```diff
@@ -41,10 +43,10 @@ public GeocodingClient getGeocodingClient(String apiKey) {
   }
 
   @Override
-  public GeocodingResult getGeocodingResult(String address) throws GeocodingException {
+  public List<GeocodingResult> getGeocodingResult(String address) throws GeocodingException {
     try {
       GeocodingResult[] response = geocodingService.address(address).await();
-      return response[0];
+      return Arrays.asList(response);
     } catch (ApiException | InterruptedException | IOException | IndexOutOfBoundsException e) {
```

Comment:
nit: I think `IndexOutOfBoundsException` can no longer be thrown?

Reply:
Yeah I believe so, thanks for catching that!

Reply:
`RESOLVED`